pmg/static/resources/javascript/committees.js

```diff
@@ -86,3 +96,55 @@ $committeeNavItem.on('click', function (e) {
 });
 
 $searchInput.on('keyup', typeAhead);
+
+// Committee detail page methods and handlers
+var filterMeetings = function($target) {
+  var filter = $target.attr('data-filter');
+  var $table = $('#m-' + filter);
+
+  $cdListTables.hide();
+  $table.fadeIn({ duration: 250, easing: 'linear' });
+
+  return filter;
+}
+
+$cdNavItem.on('click', function(e) {
+  e.preventDefault();
+  $(this).tab('show');
+
+  $cdNavMobileSelect.val($(e.target).attr('data-target'));
+});
+
+$cdNavMobileSelect.on('change', function(e) {
+  e.preventDefault();
+
+  var $selected = $('option:selected',this);
+  $selected.tab('show');
+  $cdNavItem.closest('li')
+    .removeClass('active')
+    .has('[data-target="' + $selected.attr('data-target') + '"]')
+    .addClass('active');
+});
+
+// Hide all but first ten meetings on load
+$cdCurrentYearTable.find('tr')
```

Comment:
Since the first tab to load is the 'last 3 months' or whatever, I don't think we need to do this?